ui/de-lib/src/main/java/org/iplantc/de/shared/DEProperties.java

```diff
@@ -211,6 +213,27 @@ public static DEProperties getInstance() {
     }
 
     /**
+     * list of properties UI needs
+     *
+     * @return list of properties
+     */
+    public List<String> getPropertyList() {
+        List<String> keys = new ArrayList<>();
+        keys.add("org.iplantc.communitydata.path");
+        keys.add("org.iplantc.pathList.fileIdentifier");
+        keys.add("org.iplantc.discoveryenvironment.muleServiceBaseUrl");
+        keys.add("org.iplantc.services.de-data-mgmt.base");
+        keys.add("org.iplantc.services.permIdRequests");
+        keys.add("org.iplantc.services.permIdRequests");
+        keys.add( "org.iplantc.services.admin.de-data-mgmt.base");
+        keys.add("org.iplantc.services.file-io.base.secured");
+        keys.add("org.iplantc.discoveryenvironment.notifications.poll-interval");
+        keys.add("org.iplantc.discoveryenvironment.contextMenu.enabled");
+        keys.add("org.iplantc.communitydata.path");
+        return keys;
```

Comment:
I have submitted another PR that relates to deleting all the code that involves KeepAliveTimer. Add variables instead of strings sounds like a good plan